### ISSUE TITLE
S19 Patch. It stopped working. Cursor goes upper left screen corner. 

### ISSUE THREAD

**Issue by POLTERGEISTD3** [DaLeberkasPepi/CubeConverter#8]
Can you please check this. It suddenly stopped working with newest patch.

**Comment by MrMikeU1** [comment]
I recently came across this, downloaded it and ran it. It works perfectly. No issues.

**Comment by DaLeberkasPepi** [comment]
> 
> 
> Can you please check this. It suddenly stopped working with newest patch.

Can you tell me if you were running TurboHUD at the same time? And if it works if you close TurboHUD before using this?

**Comment by POLTERGEISTD3** [comment]
Hi. Leberkäse. Sorry that i didnt come back at all. I saw it, wanted answer and then other things kept me busy and i forgot. 

So. After i wrote it to you. It suddenly worked again. I even asked all clanies. They had no issues too. Yes. Thud runs like all the time. 

BUT there was definetely an issue. I used it in S18 everyday. No issues. Then first time after patch i had this issue. I dont know if i started Thud later or before or if it was running at all at this time. Most probably it was already running.  Maybe that helps. 

And after that. The error stayed away. Maybe it was my pc ? but never before i had something like this. And other apps would go upper left corner too. Not just cubeconverter. Anyway. Right now it works. And it worked everytime after this one single time... 

Thabnk you for looking into this. We all love this little script you did. 
Bis bald :)
